README.md

```diff
@@ -186,6 +186,9 @@ if using docker, keep the container after the build.
 --make-exploded-image
 creates an exploded image (useful for codesigning jmods). Use --assemble-exploded-image once you have signed the jmods to complete the packaging steps.
 
+--custom-cacerts <true|false>
```

Comment:
Should this be really called use-mozilla-certs?

Reply:
Doesn't really matter to me. Before I make the change: What if we ever decide to include another certificate?